shadow-dom/event-composed-path-with-related-target.html

```diff
@@ -134,7 +136,9 @@
 test(() => {
   let n = createTestTree(test4);
   let log = dispatchEventWithLog(n, n.host1, new FocusEvent('my-focus', { bubbles: true, composed: true, relatedTarget: n.target }));
-  assert_event_path_equals(log, []);
```

Comment:
This change needs to reverted if we change the spec with your proposal to trim the event path to `[]` when the related target is inside the target's shadow tree but we can land this as is for now.